### PR TITLE
Fixes/tweaks wormholes event and portals

### LINKED ISSUE #30236: Space-Time anomalies aka wormholes don't teleport you? (Random Event)
roundid 75308

this was before the announcement was actually made

### PR DESCRIPTION
**Pull request by silicons**
:cl:
rscadd: Wormhole event wormholes now actually teleport you.
bugfix: portals now actually teleport on click.
/:cl:

Fixes #30236

### CODE REVIEW

**Comment by kevinz000** [comment]
wait think i did this the wrong way

**Comment by DoctoreMad** [comment]
Well if you tested this you'd know if you did it the wrong way.

**Comment by MoreRobustThanYou** [comment]
FINISH TECHWEBS REEEEE

**Comment by Shadowlight213** [comment]
One way is better. Having it be two way means people can check to see if that portal leads to the armory without the risk of it actually leading to space and getting stuck.

**Comment by kevinz000** [comment]
@Shadowlight213 yep already made one way

**Comment by YPOQ** [comment]
Fixes #30236

File: code/game/objects/effects/portals.dm [hunk 1]
```diff
@@ -40,10 +40,11 @@
 
 /obj/effect/portal/attackby(obj/item/W, mob/user, params)
 	if(user && Adjacent(user))
+		user.forceMove(get_turf(src))
```
Comment (lzimann) [inline code/game/objects/effects/portals.dm]: Why is this forceMove being added exactly? This will end up calling `teleport` twice.

Comment (kevinz000) [reply code/game/objects/effects/portals.dm]: uhhh hold on im gonna go look something up i believe this was supposed to fix something hm

**Pull request merged**